src/datreant/core/tests/__init__.py

```diff
@@ -1,15 +0,0 @@
-# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; -*-
-# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
-#
-# datreant.tests
-
-"""
-datreant.tests --- unit tests for datreant
-=============================================================
-
-"""
-# Bring some often used objects into the current namespace
-from . import test_collections
```

Comment:
http://python-notes.curiousefficiency.org/en/latest/python_concepts/import_traps.html

Pycharm IDE doesn't liked it while running tests .-)